packages/main/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const { mergeWithRules } = require("webpack-merge");
 const singleSpaDefaults = require("webpack-config-single-spa-react");
 const path = require("path");
 const webpack = require("webpack");
+const DotenvWebpackPlugin = require("dotenv-webpack");
```

Comment:
remove this require for the dotenv plugin

Reply:
@ickynavigator Done boss